generators/server/templates/src/main/java/package/config/_MicroserviceSecurityConfiguration.java

```diff
@@ -101,6 +104,11 @@ public SecurityEvaluationContextExtension securityEvaluationContextExtension() {
     @Inject
     JHipsterProperties jHipsterProperties;
 
+    <%_ if (hibernateCache !== 'hazelcast') { _%>
```

Comment:
are you sure this has to be bound to hazelcast? the issue currently happens only, because spring loads hazelcast long enough to have the client fetched services onetime....

with this hack you just force to do this twice at this point, so the second request will have already service loaded

I think, if we do this, then for all....

one other consideration is to make this "token-key-fetch" thing optional for some profile, because this behavior is making a lot of trouble, not only that service fetch thing...